desktop_version/src/Entity.cpp

```diff
@@ -11,6 +11,8 @@
 #include "Script.h"
 #include "UtilityClass.h"
 
+#include <cmath>
```

Comment:
The cmath/fabs patches make sense to me, this can be submitted separately.

Reply:
The game doesn't compile on CentOS 7 without this. I wasn't sure how to split this into a second PR and have the Git history still make sense.

Reply:
What I do is make one PR with the separate change, and branch off of that PR to submit a second PR, and label that the second PR depends on the first one.

Anyways, we should be using `SDL_abs()` in `Entity.cpp` instead of `std::abs()`.

Reply:
The correct function is SDL_fabsf, for what it's worth. std::fabs is overloaded.